requirements.txt

```diff
@@ -1,16 +1,16 @@
-awkward~=2.0.0
+awkward~=2.1.0
```

Comment:
Thank you for these updates. They eventually happy via the dependabot but it does not prevent us from acting when we find it relevant :-).